zcash_client_backend/src/data_api.rs

```diff
@@ -509,24 +536,34 @@ pub trait WalletWrite: WalletRead {
     /// The type of identifiers used to look up transparent UTXOs.
     type UtxoRef;
 
-    /// Tells the wallet to track the next available account-level spend authority, given
-    /// the current set of [ZIP 316] account identifiers known to the wallet database.
+    /// Tells the wallet to track the next available account-level spend authority, given the
+    /// current set of [ZIP 316] account identifiers known to the wallet database.
+    ///
+    /// Returns the account identifier for the newly-created wallet database entry, along with the
+    /// associated [`UnifiedSpendingKey`].
     ///
-    /// Returns the account identifier for the newly-created wallet database entry, along
-    /// with the associated [`UnifiedSpendingKey`].
+    /// If a birthday height is provided and that height is below the current chain tip, this
```

Comment:
I agree; a separate `recover_wallet` is probably a good idea, as it can correctly set scan ranges. However, since support for more than a single account is technically out of scope for the v2.0.0 SDKs, I don't think we should worry about it right now.